python/smqtk/utils/bit_utils.py

```diff
@@ -106,15 +131,18 @@ def bit_vector_to_int(v):
     :rtype: int
 
     """
-    c = 0L
-    for b in v:
-        c = (c * 2L) + int(b)
-    return c
+    if not hasattr(v, 'data') or v.dtype == numpy.dtype('object'):
+        c = 0L
+        for b in v:
+            c = (c * 2L) + int(b)
+        return c
+    else:
+        return _bit_vector_to_int_fast(v)
```

Comment:
Can numba (python module) jit compile this function?

Reply:
It can, but in investigating this more closely, I noticed that my earlier benchmarking hadn't taken into account the numba compilation time. Redoing my tests and ensuring that numba had already done its thing before the benchmarking started, I see that the numba-compiled for-loop outperforms _bit_vector_to_int_fast for this case (bit vectors <64 bits). I'll update this accordingly.